src/cmap/conn/mod.rs

```diff
@@ -61,6 +61,7 @@ pub(crate) struct Connection {
     /// currently checked into the pool, this will be None.
     pub(super) pool: Option<Weak<ConnectionPoolInner>>,
 
+    commmand_executing: bool,
```

Comment:
for future readers of the code, could we have a docstring indicating the purpose of this value / why it's necessary?

Reply:
Added